routing/pathfind_test.go

```diff
@@ -518,10 +522,13 @@ func createTestGraphFromChannels(testChannels []*testChannel) (*testGraphInstanc
 			if testChannel.Node1.MaxHTLC != 0 {
 				msgFlags |= lnwire.ChanUpdateOptionMaxHtlc
 			}
-			var channelFlags lnwire.ChanUpdateChanFlags
+			channelFlags := lnwire.ChanUpdateChanFlags(0)
```

Comment:
reason for this change? it looks equivalent?

Reply:
I guess no reason. reverted.